src/chartserver/handler_repo_test.go

```diff
@@ -1,7 +1,6 @@
 package chartserver
 
 import "testing"
-
```

Comment:
Why removing this line 4 here? Is it deleted by `make lint`?  Maybe we should keep it.

Reply:
Essentially, we would like there is an empty line between `import` and `func TestGetIndexFile`. If this  line 4 is deleted automatically by `make lint`,  maybe we can try to change 
```
import "testing"

// Test get /index.yaml
func TestGetIndexFile(t *testing.T) {
...
}
```
to 
```
import (
    "testing"
)

// Test get /index.yaml
func TestGetIndexFile(t *testing.T) {
...
}
```
to  ensure there is an empty line between `import` and its subsequent code.

Reply:
Got it, I will try to update.

Reply:
Now all linters do not config the one-line import, they will keep it as they were. I recommend that we keep the same with them.

The emply line is necessary, I will update them all but keep one-line import as they were.

Reply:
Cool, thanks a lot!